app/controllers/tas_controller.rb

```diff
@@ -51,13 +52,22 @@ def download_ta_list
         [ta.user_name,ta.last_name,ta.first_name,ta.email]
       end
       format = 'text/csv'
-    when 'xml'
-      output = tas.to_xml
-      format = 'text/xml'
+    when 'yml'
+      output = []
+      count = 0
+      tas.all.each do |ta|
+        count += 1
+        output.push([{ "TA_#{count}" => [[{ 'Username' => [ta.user_name] }],
```

Comment:
I'd like the format here to be different. `output` should eventually be a list of Hashes, where each element is of the form
```
{
  user_name: ta.user_name,
  last_name: ta.last_name,
  first_name: ta.first_name,
  email: ta.email
}
```

This should result in a yaml output of

```
- user_name: blah
  last_name: blah_first
  first_name: blah_last
  email: 123@gmail.com
- user_name: blah2
  last_name: blah2_first
  first_name: blah2_last
  email: 234@gmail.com
```